ServiceDocumentSampleComponents/serviceDocumentSampleComponents/force-app/main/default/lwc/mediumHeaderText/mediumHeaderText.js-meta.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>59.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__ServiceDocument</target>
+    </targets>
+    <targetConfigs>
+        <targetConfig targets="lightning__ServiceDocument">
+            <supportedFormFactors>
+                <supportedFormFactor type="Large" />
```

Comment:
Add supportedFormFactor for Small just in case

Reply:
Yea done.